tests/integration/conformance/main_test.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in conformance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package conformance
+
+import (
+	"os"
+	"path"
+	"testing"
+
+	"istio.io/istio/pkg/test/conformance"
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/environment"
+	"istio.io/istio/pkg/test/framework/components/istio"
+)
+
+func loadCases() ([]*conformance.Test, error) {
+	wd, err := os.Getwd()
+	if err != nil {
+		return nil, err
+	}
+	p := path.Join(wd, "cases")
```

Comment:
Doesn't it need to be "testdata" to play nicely with "go test" command?

Reply:
That is needed if you want go tools to ignore data files. Which is probably very useful if your data files are .go files. This is not the case here.

Reply:
Hm I thought I saw somewhere test cache not working properly when using data files from other directories, but cannot reproduce now. Maybe it was a different kind of issue. Though per [rsc's comment](https://github.com/golang/go/issues/22583#issuecomment-342963522) there are still some minor advantages to the well-known testdata.